mod/contacts.php

```diff
@@ -554,7 +554,7 @@ function contacts_content(App $a)
 		// tabs
 		$tab_str = contacts_tab($a, $contact_id, 2);
 
-		$lost_contact = (($contact['archive'] && $contact['term-date'] > NULL_DATE && $contact['term-date'] < datetime_converL10n::t('', '', 'now')) ? L10n::t('Communications lost with this contact!') : '');
```

Comment:
Ups ... have you had a look at other possible false replacements of ```t(```?

Reply:
How do you suggest I check this when it isn't failing?

Reply:
This **should** be the only one that slipped through the cracks.. I was as careful as I could be when searching through the million occurrences of t(